lib/cocoapods/generator/embed_frameworks_script.rb

```diff
@@ -34,6 +34,11 @@ def save_as(pathname)
       # @return [String] The contents of the embed frameworks script.
       #
       def script
+        codesign_bg = ''
+        parallel_codesign = ENV['PARALLEL_CODE_SIGN'] == 'true'
```

Comment:
Looks like we use `COCOAPODS`, `COCOA_PODS`, and `CP`. I'll use the first one, since that's what we appear to use the most :)